plugin.audio.sverigesradio/resources/language/Swedish/strings.xml

```diff
@@ -14,9 +14,12 @@
     <string id="32000">Inställningar</string>
```

Comment:
strings.xml is not compatible with Matrix, support has been fully removed. Please adapt the new gettext format:

https://kodi.wiki/view/Language_support

Reply:
Any chance of merging anyways, seeing as the current version is broken (poor error handling on my part)